src/test/java/org/antonkhmarun/tests/CreatingAccountTest.java

```diff
@@ -0,0 +1,165 @@
+import org.openqa.selenium.support.ui.Select;
+
+import java.text.ParseException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreatingAccountTest extends BaseTest{
+
+    public static PersonalInformation personalInformation;
+    public static Authentication authentication;
+    private final Faker faker = new Faker();
+
+    @BeforeEach
+    public void beforeEachTest() {
+        personalInformation = new PersonalInformation(driver);
+        authentication = new Authentication(driver);
+        driver.get(ConfProperties.getProperty("loginPage"));
+    }
+
+    @Test
+    public void createAccountTest() throws ParseException {
+        authentication.inputEmailCreate(faker.internet().emailAddress());
+        authentication.clickCreateAccountBtn();
+
+        personalInformation.clickRandomRadioBtnMrMrs();
+
+        String firstName = faker.name().firstName();
+        String lastName = faker.name().lastName();
+        fillAllFieldsWithRandomValidData(firstName, lastName);
+
+        personalInformation.clickSubmitAccountBtn();
+
+        assertEquals(firstName + " " + lastName, authentication.getAccountName());
+
+        authentication.logout();
+    }
+
+    @Test
+    public void createAccountWithoutCustomerFirstName() throws ParseException {
+        authentication.inputEmailCreate(faker.internet().emailAddress());
+        authentication.clickCreateAccountBtn();
+
+        personalInformation.clickRandomRadioBtnMrMrs();
+
+        fillAllFieldsWithRandomValidData();
+
+        personalInformation.cleanCustomerFirstName();
+        personalInformation.clickSubmitAccountBtn();
+
+        String expectedAlertTitle = "There is 1 error";
+        String expectedAlertMessage = "firstname is required.";
+
+        assertEquals(expectedAlertTitle, personalInformation.getAlertDangerTitle());
+        assertEquals(List.of(expectedAlertMessage), personalInformation.getListAlertDangerMessages());
+    }
+
+    @Test
+    public void createAccountWithoutCustomerFirstAndLastName() throws ParseException {
+        authentication.inputEmailCreate(faker.internet().emailAddress());
+        authentication.clickCreateAccountBtn();
+
+        personalInformation.clickRandomRadioBtnMrMrs();
+
+        fillAllFieldsWithRandomValidData();
+
+        personalInformation.cleanCustomerFirstName();
+        personalInformation.cleanCustomerLastName();
+        personalInformation.clickSubmitAccountBtn();
+
+        String expectedAlertTitle = "There are 2 errors";
+        String expectedAlertMessage1 = "lastname is required.";
+        String expectedAlertMessage2 = "firstname is required.";
+
+        assertEquals(expectedAlertTitle, personalInformation.getAlertDangerTitle());
+        assertEquals(List.of(expectedAlertMessage1, expectedAlertMessage2), personalInformation.getListAlertDangerMessages());
+    }
+
+    private void fillAllFieldsWithRandomValidData() throws ParseException {
```

Comment:
Those method better to move to the page class. In such case it can be reused by other tests in other test classes